templates/advantage/index.html

```diff
@@ -401,6 +517,72 @@ <h2>Plans for enterprise use</h2>
   }
 </script>
 
+
+<script>
```

Comment:
This script is in two places now (the other is /openstack/compare) - I think it'd be worth adding it to /js/utils and including a link to it on both pages.

Reply:
Good point, done